aes/src/ni.rs

```diff
@@ -71,36 +44,25 @@
 //! - [Intel AES-NI whitepaper](https://software.intel.com/sites/default/files/article/165683/aes-wp-2012-09-22-v01.pdf)
 //! - [Use of the AES Instruction Set](https://www.cosic.esat.kuleuven.be/ecrypt/AESday/slides/Use_of_the_AES_Instruction_Set.pdf)
 
-#![no_std]
-#![doc(
-    html_logo_url = "https://raw.githubusercontent.com/RustCrypto/meta/master/logo.svg",
-    html_favicon_url = "https://raw.githubusercontent.com/RustCrypto/meta/master/logo.svg"
-)]
-#![warn(missing_docs, rust_2018_idioms)]
-
-pub use cipher;
-
-#[cfg(feature = "ctr")]
-pub use cipher::stream;
-
-#[cfg(not(feature = "nocheck"))]
-mod target_checks;
```

Comment:
It's a bit unclear to me what should happen to this module in a merged crate. Should we remove it?

Reply:
Yes, the feature and the module will be of no use in the unified crate.

Reply:
Removed in fa79e60